### PR TITLE
implemented a function that displays the instructions to the user

### LINKED ISSUE #2: Add a function that defines game instructions
We need to implement a function that is responsible for displaying game instructions to players at the beginning of the game:

- initialize `instuction.py` file
- implement a function that displays the instructions to the user

### PR DESCRIPTION
**Pull request by SBMOYO**

### CODE REVIEW

File: instuction.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def main():
+    first_user = input("Player 1 name: ")
+    second_user = input("Player 2 name: ")
+    print(f"{first_user} has been assigned a symbol (X) and the {second_user} has been assigned a symbol (O) ")
+    print(f"The game will be played between {first_user} and {second_user}")
+    print("the objective is to make a straight line of three using your assigned symbols by choosing correct numbers and win!")
```
Comment (ClaytonSiby) [inline instuction.py]: You can use a multiple-line syntax to create this string and use only one print statement to display the output. Please refer to [this](https://www.w3schools.com/python/python_strings.asp).

**Review by ClaytonSiby — COMMENTED** [review]
Looking good, I left a suggestion, please address that and send me a re-review request once done so I can take a look again.

File: instuction.py (hunk 1 above)
Comment (SBMOYO) [reply instuction.py]: ok sure let me work on it

**Review by ClaytonSiby — APPROVED** [review]
Looking good 👍🏽

**Pull request merged**